modules/graphql/index.js

```diff
@@ -8,7 +8,9 @@ module.exports = (ctx) => {
 
 function _request(ctx) {
   return function (params, cb) {
-    params.uri = 'https://api_test.dronedeploy.com/graphql';
+    const env = config.env || global.NODE_ENV || "prod";  // default to prod
+    const envUrlPart = env === "prod" ? "" : "_" + env;
```

Comment:
instead of doing this, can we not just make the endpoint part of the config so it's calculated once?